### PR TITLE
eliminate google-specific oncall group and page

### PR DESCRIPTION
**Pull request by BenTheElder**
These haven't made sense anymore for a long time now, we will not migrate this to community infra and will instead go ahead and replace https://go.k8s.io/oncall with a tombstone with pointers to alternate contact points 

Follow-up PRs to this and other repos will:
- Remove the test-infra-oncall slack group (https://github.com/kubernetes/community/pull/7985)
- Drop the oncall.html upload job (after this version uploads, I'm going to go disable TTL on the underlying bucket as well)

But we need this PR first.

I plan to send a short notice to dev@kubernetes.io and relevant slack groups to highlight this change.

### CODE REVIEW

File: maintenance/oncall.html [hunk 1]
```diff
@@ -1,106 +1,10 @@
 <!DOCTYPE html>
```
Comment (BenTheElder) [inline maintenance/oncall.html:1]: rendered:
Screenshot 2024-07-23 at 2 36 09 PM

**Review by michelle192837 — APPROVED** [review]
Thanks Ben!

/hold

In case others want to take a look, unhold at will.

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/community/pull/7985 and https://github.com/kubernetes/test-infra/pull/33084 next

**Pull request merged**